web-ui/src/main/resources/catalog/views/default/templates/recordView.html

```diff
@@ -322,7 +322,10 @@ <h2 data-translate="">technicalInformation</h2>
             <th data-translate="">crs</th>
             <td>
               <ul>
-                <li data-ng-repeat="r in ::mdView.current.record.crs">{{r}}</li>
+                <li data-ng-repeat="r in ::mdView.current.record.crs">
+                  <span ng-if="r.name != ''">{{ r.name }} ({{ r.code }})</span>
```

Comment:
We don't need to add crs as a JSON field to parse JSON and be able to access object props ? cf. https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/catalog/CatalogService.js#L472